src/wallet/rpcwallet.cpp

```diff
@@ -1164,6 +1175,10 @@ UniValue ListReceived(const UniValue& params, bool fByAccounts)
             if (!ExtractDestination(txout.scriptPubKey, address))
                 continue;
 
+            if (fFilterAddress && !(filterAddress == address)) {
```

Comment:
I thought if all variants accepted the operator the variant would accept it, in which case adding it to `CNoDestination` would be enough, but maybe I'm off on that one.